build.gradle

```diff
@@ -17,20 +17,15 @@ buildscript {
         classpath "com.netflix.nebula:gradle-aggregate-javadocs-plugin:3.0.1"
         classpath "com.google.protobuf:protobuf-gradle-plugin:0.8.8"
         classpath "io.franzbecker:gradle-lombok:3.2.0"
-        /*
-         * TODO: Upgrading to 1.3.1 caused problems
-         * A problem was found with the configuration of task ':genie-ui:npmInstall'.
-         * > File '{somePath}/genie/genie-ui/package-lock.json' specified for property '$2' does not exist.
-         */
-        classpath "com.moowork.gradle:gradle-node-plugin:1.2.0"
+        classpath "com.netflix.nebula:nebula-node-plugin:1.3.0"
         /*
          * TODO: Failed when upgrading to 2.0.0. Looks like a Guava version problem (surprise)
          * Execution failed for task ':genie-agent:spotbugsIntegTest'.
          * > Failed to run Gradle SpotBugs Worker
          *   > Could not create an instance of type com.github.spotbugs.internal.spotbugs.SpotBugsExecutor.
          *      > com.google.common.collect.ImmutableList.builderWithExpectedSize(I)Lcom/google/common/collect/ImmutableList$Builder;
          */
-        classpath "gradle.plugin.com.github.spotbugs:spotbugs-gradle-plugin:1.6.9"
+        classpath "com.github.spotbugs:spotbugs-gradle-plugin:3.0.0"
```

Comment:
If this fixes the issue we can probably get rid of the big comment block above this.